CONTRIBUTING.md

```diff
@@ -77,50 +77,52 @@ Then run the whole integ test suite against Cadence+Temporal service by this com
 
-NOTE: For Temporalite following the [instruction](https://github.com/temporalio/temporalite). If you see
-error `error setting up schema`, try use command `temporalite start --namespace default -f my_test.db` instead to start.
 
-NOTE: Temporal docker compose uses http://localhost:8080/ but Temporalite uses http://localhost:8233/ for WebUI
+### Option 2: Run with your own Temporal service
 
-Assuming you already have a Temporal service :
+First of all, you need a Temporal service if you haven't had it:
 
-1. Make sure you have a default namespace
+Option 1 (recommended): use [Temporal CLI](https://github.com/temporalio/cli) -- `temporal server start-dev`
 
-```shell
-tctl --ns default n re
-```
+Option 2: use [temporal docker-compose](https://github.com/temporalio/docker-compose)
 
-2. Make sure you have registered system search attributes required by iWF server
 
-```shell
-tctl adm cl asa -n IwfWorkflowType -t Keyword
-tctl adm cl asa -n IwfGlobalWorkflowVersion -t Int
-tctl adm cl asa -n IwfExecutingStateIds -t Keyword
+Assuming you are using `default` namespace:
+
+1. Make sure you have registered system search attributes required by iWF server
 
+```shell
+  temporal  operator search-attribute  create -name IwfWorkflowType -type Keyword
+  temporal  operator search-attribute  create -name IwfGlobalWorkflowVersion -type Int 
+  temporal  operator search-attribute  create -name IwfExecutingStateIds -type KeywordList 
 ```
 
 3. For `persistence_test.go` integTests, you need to register below custom search attributes.
 
 ```shell
-tctl adm cl asa -n CustomKeywordField -t Keyword
-tctl adm cl asa -n CustomIntField -t Int
-tctl adm cl asa -n CustomBoolField -t Bool
-tctl adm cl asa -n CustomDoubleField -t Double
-tctl adm cl asa -n CustomDatetimeField -t Datetime
-tctl adm cl asa -n CustomStringField -t text
+  temporal  operator search-attribute  create -name CustomKeywordField -type Keyword
+  temporal  operator search-attribute  create -name CustomIntField -type Int
+  temporal  operator search-attribute  create -name CustomBoolField -type Bool
+  temporal  operator search-attribute  create -name CustomDoubleField -type Double
+  temporal  operator search-attribute  create -name CustomDatetimeField -type Datetime
+  temporal  operator search-attribute  create -name CustomStringField -type Text
 ```
 
 4. If you run into any issues with Search Attributes registration, use the below command to check the existing Search
-   attributes:`tctl adm cl get-search-attributes`
+   attributes:`temporal operator search-attribute list`
 
-Then run the whole integ test suite against Cadence+Temporal service by this command:
+Then run the whole integ test suite against Temporal service by this command:
 
 `make temporalIntegTests`
 
+To run the server, if you are in an IDE, you can run the main function in `./cmd/main.go` with argument `start`.
+
+Alternatively, you could just build the binary by `make bins` and then run `./iwf-server start`
```

Comment:
These two sentences here are the same as the line 80 and 82?

Reply:
yeah that's not nice. I just moved them to the same place.